script/Arcueid/Day 6/06_00_ARC06_1.txt

```diff
@@ -414,7 +414,7 @@ It seems that just the pain of trying to walk is enough to cause my body to reac
 [sha:529165c4e14e25031fd833e74812a1aa44073462]{
 -- Page 32, Offset 14002.
 -- 先ほどはああ申しましたが、秋葉さまには私から……」
-You cried out a little earlier, but if I were to put in a word with Akiha-sama..."
+You let out a moun a little earlier, but if I were to put in a word with Akiha-sama..."
```

Comment:
- Spelling is moan
- I don't think _moaning_ is what he did, she's referring to his shriek when he was startled by her presence IIRC